src/RecoveryServices/RecoveryServices.SiteRecovery/Properties/Resources.Designer.cs

```diff
@@ -520,7 +520,18 @@ internal class Resources {
                 return ResourceManager.GetString("NicNotFoundInVM", resourceCulture);
             }
         }
-        
+
+        /// <summary>
+        ///   Looks up a localized string similar to IP config &quot;{0}&quot; not found in VM Nic..
+        /// </summary>
+        internal static string IPConfigNotFoundInVMNic
```

Comment:
dont see the resx file changes for this new warning string

Reply:
added